Sources/StreamChatUI/ChatChannelList/ChatChannelListVC.swift

```diff
@@ -124,6 +123,10 @@ open class ChatChannelListVC: _ViewController,
             return
         }
 
+        guard collectionView.isTracking || collectionView.isDecelerating else {
```

Comment:
Seems like this same check is duplicated in ChatChannelVC, what about creating an extension over UICollectionView?

Reply:
one VC uses tableview and the other collectionview

Reply:
Both are UIScrollView under the hood